package.json

```diff
@@ -30,6 +30,7 @@
   "homepage": "https://github.com/wix-incubator/tpa-style-webpack-plugin",
   "dependencies": {
     "@ctrl/tinycolor": "^2.2.1",
+    "@wix/tpa-settings": "^2.0.260",
```

Comment:
We should consider migrating `tpa-style-webpack-plugin` under `yoshi` monorepo.

For now `@wix/tpa-settings` is a private package, so Travis tests could not install this package

Reply:
i hope we don't have 3rd party users because if so, we will break them.
we need to verify with Oron that wix-labs have access to it.